src/cocotb/share/lib/embed/gpi_embed.cpp

```diff
@@ -139,16 +82,41 @@ extern "C" COCOTB_EXPORT void _embed_init_python(void) {
         }
     }
 
-    to_python();
-    // must set program name to Python executable before initialization, so
-    // initialization can determine path from executable
-    set_program_name_in_venv();
-    Py_Initialize(); /* Initialize the interpreter */
-    PySys_SetArgvEx(1, argv, 0);
-
-    /* Swap out and return current thread state and release the GIL */
-    gtstate = PyEval_SaveThread();
```

Comment:
This variable was not used for anything besides preventing re-initialization, so it was swapped for a simple bool.